frontend/src/components/Chat/Chat.jsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+import { getRooms } from '../../api.js';
+import { MessageForm } from '../MessageForm/MessageForm.jsx';
+import { MessageList } from '../MessageList/MessageList.jsx';
+import { NameChooser } from '../NameChooser/NameChooser.jsx';
+import { Rooms } from '../Rooms/Rooms.jsx';
+import './Chat.css';
+import { socket } from '../../webSocket.js';
+
+<h1 className="title">Chat application</h1>;
+export function App() {
+  const [messages, setMessages] = useState([]);
+  const [rooms, setRooms] = useState([]);
+  const [room, setRoom] = useState(null);
```

Comment:
it is not a mistake, but it decreases readability, try to avoid using similar names with a difference in one letter (room, rooms, etc.)